node/libs/roles/src/proto/validator/messages.proto

```diff
@@ -72,9 +72,8 @@ message ReplicaNewView {
 }
 
 message LeaderProposal {
```

Comment:
Add reserved field.

Reply:
Missed it.

Reply:
Wait. It shouldn't be necessary. This is a new type. I only added it in the previous PR, and that's not merged into main yet.